app/controllers/clear_cms/contents_controller.rb

```diff
@@ -126,6 +128,38 @@ def import
       redirect_to({:action=>:index}, notice: 'Content imported successfully.')
     end
 
+    def csv_importer
+    end
+
+    def csv_import
+      headers = CSV.read(params[:dump][:file].path, headers: true).headers
+      attributes_updated = headers & ClearCMS::Content.csv_allowable_attributes
+        CSV.foreach(params[:dump][:file].path, headers: true) do |row|
+          begin
+            product = ClearCMS::Content.find(row["id"])
+          rescue Exception => e
```

Comment:
You shouldn't ever really rescue just generic Exceptions in ruby...you can accidentally rescue exceptions that should be bubbling up, and in this case you want to rescue a specific, ActiveRecord::RecordNotFound exception.  

http://www.mikeperham.com/2012/03/03/the-perils-of-rescue-exception/